src/main/java/chess/controller/web/WebChessController2.java

```diff
@@ -0,0 +1,62 @@
+package chess.controller.web;
+
+import chess.domain.game.BoardFactory;
+import chess.domain.game.Game;
+import chess.service.LoadService;
+import chess.service.MoveService;
+import chess.service.StartService;
+import spark.ModelAndView;
+import spark.Request;
+import spark.Response;
+import spark.template.handlebars.HandlebarsTemplateEngine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static spark.Spark.get;
+import static spark.Spark.post;
+
+public class WebChessController2 {
+    private final StartService startService;
+    private final MoveService moveService;
+    private final LoadService loadService;
+    private Game game;
+
+    public WebChessController2(StartService startService, MoveService moveService, LoadService loadService) {
+        this.startService = startService;
+        this.moveService = moveService;
+        this.loadService = loadService;
+    }
+
+    public void run() {
+        get("/", this::welcomePage);
+        get("/start", this::start);
+        post("/chess", this::move);
+        get("/load", this::load);
+    }
+
+    private Object welcomePage(Request request, Response response) {
+        Map<String, Object> model = new HashMap<>();
+        return new HandlebarsTemplateEngine().render(new ModelAndView(model, "index.html"));
+    }
+
+    private Object start(Request request, Response response) {
+        init();
+        return startService.start(game);
+    }
+
+    private Object move(Request request, Response response) {
+        String command = request.queryParams("command");
+        return moveService.move(game, command);
+    }
+
+    private Object load(Request request, Response response) {
+        init();
```

Comment:
어차피 새로 생성하는데 필드로 들고있을 이유가 없을 것 같아요 :) 
다음 레벨 진행하면서 `Thread safe` 에 관해서 한번 찾아보세요! 
```suggestion
        Game game = new Game(BoardFactory.create());
```

Reply:
move의 경우를 위해서 game을 필드로 가지고 있었던 것 같아요!
move의 경우에도 game을 새로만들어 사용하기 위해서는 load를 해오고 나서 move명령어를 실행시키고 오류가 없을 경우만 디비에 추가하는 방식을 적용시켜야 할 것 같니요 ㅜ